<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserWindow</class>
 <widget class="QDialog" name="EditUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XKOMX</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbName">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbSurname">
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSurname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbCity">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCity"/>
     </item>
     <item>
      <widget class="QLabel" name="lbPostalCode">
       <property name="text">
        <string>Postal code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePostalCode">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbAddress">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lbCountry">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCountry"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnChangeInfo">
     <property name="text">
      <string>Change user information</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="lbOldPass">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>113</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Old password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOldPass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="lbNewPass">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>113</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>New password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leNewPass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="lbConfirmPass">
       <property name="maximumSize">
        <size>
         <width>113</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Confirm new password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leConfirmPass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnChangePassword">
     <property name="text">
      <string>Change password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Purchase history</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
